mobile/src/common/radio-buttons-group.component.tsx

```diff
@@ -0,0 +1,175 @@
+/******************************************************************************
+ * Copyright (c) Arcadia, Inc. All rights reserved.
+ ******************************************************************************/
+
+import React from 'react';
+import { TextStyle, TouchableOpacity, View, ViewStyle } from 'react-native';
+import { StyledText } from '../override/styled-text';
+import { radioButtonsGroupStyles } from './radio-buttons-group.styles';
+
+//============================================================================
+export interface RadioButton {
+    selectionIndex: number;
+    label: string;
+
+    labelStyle?: TextStyle;
+    color?: string;
+    disabled?: boolean;
+    layout?: string;
+    selected?: boolean;
+    size?: number;
+}
+
+//============================================================================
+interface RadioButtonProps extends RadioButton {
+    onPress: (label: string) => void;
+}
+
+//============================================================================
+class RadioButtonComponent extends React.Component<RadioButtonProps> {
```

Comment:
just `RadioButton`

Reply:
there is already interface with this name